server/Routes/Routes.js

```diff
@@ -51,6 +54,63 @@ Router.post("/register", async (req, res) => {
   }
 });
 
+
+// route for login using username or email
+
+Router.post('/signin', async (req,res) => {
```

Comment:
Add a controller file and implement the router logic there. Keep Router.js clean with just the component

Reply:
Implemented controllers to the routes.